docker-wrapper.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Place holder script to configure cockpit environment
+#
+# To prevent this thing to happen on every start, we create a mark file.
+# Otherwise, every time a container is stopped and restarted, the env.sh file will be updated.
+if [ ! -f docker-mark.txt ]; then
+# put any setup action to be run once here
+	echo "Indicate the Petals configuration was already updated for this container." > docker-mark.txt
+fi
+
+# Launch Petals-cockpit
+/opt/petals-cockpit/petals-cockpit.sh
```

Comment:
A very minor thing: please, add an empty at the end of the file (Github generally does not like it).